### PR TITLE
TS, Webpack, and development environment setup complete

### PR DESCRIPTION
**Pull request by donjewey**
Integrated webpack, typescript, husky, prettier, and eslint into the project, as well as all of the necessary scripts into `package.json`

### CODE REVIEW

**Comment by donjewey** [comment]
#1 Issue completed, except for file structure. Need to merge with backend integration PR to complete outlying file structure.

**Review by milos381 — APPROVED** [review]

**Review by evanmcneely — CHANGES_REQUESTED** [review]
1. Add these properties to the tsconfig file: "strictNullChecks" true and "noImplicitAny" true

**Review by shalarewicz — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        "dotenv": "^16.0.1",
+        "dotenv-webpack": "^7.1.0",
+        "html-loader": "^3.1.0",
+        "html-webpack-plugin": "^5.5.0",
+        "mini-css-extract-plugin": "^2.6.0",
+        "react": "^18.1.0",
+        "react-dom": "^18.1.0",
+        "react-router": "^6.3.0",
+        "react-router-dom": "^6.3.0",
+        "webpack": "^5.72.1",
```
[the remark this answers is not in the document]
Comment (donjewey) [reply package.json:41]: Done!

**Review by evanmcneely — APPROVED** [review]
We may still have to configure prettier and eslint after this but we can do that separately.

**Review by shalarewicz — APPROVED** [review]

**Pull request merged**